honeybee_schema/radiance/properties.py

```diff
@@ -0,0 +1,94 @@
+"""Properties Schema"""
+from pydantic import Field, constr, validator, root_validator
+from typing import List, Union
+
+from .modifier import Plastic, Glass, BSDF, Glow, Light, Trans, Void
+from .modifierset import ModifierSet, ModifierSetAbridged
+from .._base import NoExtraBaseModel
+
+# Unioned Modifier Schema objects defined for type reference
+_REFERENCE_UNION_MODIFIERS = Union[Plastic, Glass, BSDF, Glow, Light, Trans, Void]
+
+
+class _PropertiesBaseAbridged(NoExtraBaseModel):
+    """Base class of Abridged Radiance Properties."""
+
+    modifier: str = Field(
+        default=None,
+        description='A string for a Honeybee Radiance Modifier.'
+        )
+
+    modifier_blk: str = Field(
+        default=None,
+        description='A string for a Honeybee Radiance Modifier to be used '
+                    'in direct solar simulations and in isolation studies (assessing'
+                    'the contribution of individual objects).'
+        )
+
+
+class ApertureRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Aperture Abridged."""
+
+    type: constr(regex='^ApertureRadiancePropertiesAbridged$') = \
+        'ApertureRadiancePropertiesAbridged'
+
+
+class DoorRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Door Abridged."""
+
+    type: constr(regex='^DoorRadiancePropertiesAbridged$') = \
+        'DoorRadiancePropertiesAbridged'
+
+
+class FaceRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Face Abridged."""
+
+    type: constr(regex='^FaceRadiancePropertiesAbridged$') = \
+        'FaceRadiancePropertiesAbridged'
+
+
+class ShadeRadiancePropertiesAbridged(_PropertiesBaseAbridged):
+    """Radiance Properties for Honeybee Shade Abridged."""
+
+    type: constr(regex='^ShadeRadiancePropertiesAbridged$') = \
+        'ShadeRadiancePropertiesAbridged'
+
+
+class RoomRadiancePropertiesAbridged(NoExtraBaseModel):
+    """Abridged Radiance Properties for Honeybee Room."""
+
+    type: constr(regex='^RoomRadiancePropertiesAbridged$') = \
+        'RoomRadiancePropertiesAbridged'
+
+    modifier_set: List[str] = Field(
```

Comment:
This is not a list. A Room can have only one `ModifierSet`. The default should be `None` here.